client/src/pages/CreateNewPage.css

```diff
@@ -0,0 +1,46 @@
+.Form {
+}
+
+.Page__heading {
+  border: 2px none black;
+  padding: 2em 8em;
```

Comment:
this padding is huge and exceeds the viewport of mobile devices. try smaller values and the whole page will fit without scrolling into an iphone display. :)))